### PR TITLE
Add check for lang attribute in <html>

### PR DESCRIPTION
**Pull request by SavvasStephanides**
The `<html>` tag should have a `lang` attribute to tell the screen reader what language the page is in. 

Evidence: https://www.w3.org/WAI/standards-guidelines/act/rules/html-page-lang-b5c3f8/

### CODE REVIEW

File: checka11y.css [hunk 1]
```diff
@@ -10,6 +10,19 @@
...
+/**
+* HTML
+*/
+
+html:not([lang]) body{
+  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
+}
+
+html:not([lang]) body::after{
+  content: "ERROR: html has no lang attribute. " !important;
```
Comment (jackdomleo7) [inline checka11y.css:22]: Sorry for being picky, could these be single quotes? I may consider configuring stylelint to the project, what do you think?

File: test/index.html [hunk 2]
```diff
@@ -31,5 +31,8 @@ <h2>Lists: All children of &lt;ul&gt; must only be &lt;li&gt;</h2>
         <div>Hello, I'm a div inside a &lt;ul&gt;</div>
       </ol>
     </section>
+    <section>
+      <h2>HTML: html tag should have a lang attribute</h2>
```
Comment (jackdomleo7) [inline test/index.html:35]: This is nice because there's no `lang` on this file so that's a win 💪

Comment (jackdomleo7) [reply test/index.html:35]: No action needed 😊

File: checka11y.css (hunk 1 above)
Comment (SavvasStephanides) [reply checka11y.css]: Not a problem at all. I'm pushed an update for this

**Pull request merged**